articles/appliance/planning/appliance-vs-public-cloud.md

```diff
@@ -0,0 +1,27 @@
+---
+section: appliance
+description: Compares features available in the Auth0 Public Cloud and the Appliance
+---
+
+# Auth0 Features Comparison
+
+While most features are available to you regardless of your deployment choice, there are some differences in what's available in the Public Cloud and what's available to those using the Appliance (more specifically, deployments hosted on-premise, in your private cloud environment, or in Auth0's Private Cloud).
+
+## Current Limitations for Customers Using the Appliance
+
+1. Multifactor Authentication (MFA) is only available for TOTP and HOTP-style factorization. Guardian and SMS-based MFA is *not* available.
```

Comment:
Can we say this instead: `Multi-factor Authentication (MFA) is only available for Google Authenticator over TOTP and HOTP style factorization;  Duo, Guardian, and SMS MFA are not currently available.`